apluslms_roman/cli.py

```diff
@@ -365,20 +435,155 @@ def build_action(context):
     return result.code
 
 
+def init_action(context):
+    cwd = getcwd()
+    try:
+        config = ProjectConfig.find_from(cwd)
+        exit(1, _("A project configuration already exists at {}".format(config.path)))
+    except FileNotFoundError:
+        pass
+
+    if context.args.project_config:
+        file_ = context.args.project_config
+        if '.' not in file_:
+            is_recognized = False
+        else:
+            name, prefix = file_.split('.', 1)
+            if (name not in ProjectConfig.DEFAULT_NAMES
+                    or prefix not in ProjectConfig.DEFAULT_PREFIXES):
+                is_recognized = False
+
+        if not is_recognized:
```

Comment:
With `roman -f roman.yml` this should raise `is_recognized` is not initialized. None of the code sets it to True.